src/simpla-collection.html

```diff
@@ -45,7 +45,7 @@
          */
         items: {
           type: Array,
-          value: []
+          value: () => []
```

Comment:
Bah, keep forgetting that. Stupid Polymer, wish it was stricter one way or the other.